app/src/common/shared/com/igalia/wolvic/utils/BitmapCache.java

```diff
@@ -32,6 +36,28 @@ public class BitmapCache {
     private Surface mCaptureSurface;
     private boolean mCapturedAcquired;
 
+    private static String bytesToHex(byte[] hash) {
+        StringBuilder hexString = new StringBuilder(2 * hash.length);
+        for (int i = 0; i < hash.length; i++) {
```

Comment:
Perhaps you could just do something like for (byte b : hash) since you don't need the index for anything.

Reply:
Updated